exp-player/addon/components/exp-frame-base.js

```diff
@@ -64,19 +64,16 @@ export default Ember.Component.extend({
             this.set('id', `${kind}-${frameIndex}`);
         }
     },
-    setupParams(params, clean) {
+    setupParams(clean) {
         // Add config properties and data to be serialized as instance parameters (overriding with values explicitly passed in)
-        params = params || this.get('frameConfig');
+       var params = this.get('frameConfig');
```

Comment:
`setupParms` is only used once [here](https://github.com/CenterForOpenScience/exp-addons/blob/4f1095706ecdae524dc6ef528a4b1c132fb95e4c/exp-player/addon/components/exp-frame-base.js#L54) and `params` is `null` so it doesn't seem necessary. Sam suggested it may be a remnant of old code.